packages/xo-server/src/api/host.mjs

```diff
@@ -19,9 +19,24 @@ const log = createLogger('xo:api:host')
 
 // ===================================================================
 
-export function setMaintenanceMode({ host, maintenance }) {
+export async function setMaintenanceMode({ host, maintenance, vmsToForceMigrate }) {
   const xapi = this.getXapi(host)
 
+  if (vmsToForceMigrate) {
+    await Promise.all(
+      vmsToForceMigrate.map(async vmUuid => {
+        const record = await xapi.getRecordByUuid('VM', vmUuid)
+        const ref = record.$ref
+        const blockedOperations = record.blocked_operations
+        await Promise.all(
+          Object.keys(blockedOperations).map(
+            async operation => await xapi.call('VM.remove_from_blocked_operations', ref, operation)
+          )
+        )
+      })
```

Comment:
I think we should only remove the 2 operations related to migration. For example we should not remove any VM that as  "start" as a blocked operation

Reply:
Done